dspace/config/crosswalks/oai/metadataFormats/oai_openaire.xsl

```diff
@@ -526,19 +526,30 @@
    <!-- datacite:rights -->
    <!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_accessrights.html -->
     <xsl:template match="doc:element[@name='dc']/doc:element[@name='rights']/doc:element" mode="datacite">
+        <xsl:variable name="rightsValue" select="doc:field[@name='value']/text()"/>
         <xsl:variable name="rightsURI">
             <xsl:call-template name="resolveRightsURI">
-                <xsl:with-param name="field" select="doc:field[@name='value']/text()"/>
+                <xsl:with-param name="field" select="$rightsValue"/>
             </xsl:call-template>
         </xsl:variable>
-        <datacite:rights>
-            <xsl:if test="$rightsURI">
-                <xsl:attribute name="rightsURI">
-                <xsl:value-of select="$rightsURI"/>
-            </xsl:attribute>
-            </xsl:if>
-            <xsl:value-of select="doc:field[@name='value']/text()"/>
-        </datacite:rights>
+        <xsl:variable name="lc_rightsValue">
+            <xsl:call-template name="lowercase">
+                <xsl:with-param name="value" select="$rightsValue"/>
+            </xsl:call-template>
+        </xsl:variable>
+        <!-- this conditions ensures what is referred in issue: #3097 -->
+        <!-- it's a solution to ensure that only values ended with "access" -->
+        <!-- can be used as datacite:rights -->
+        <xsl:if test="ends-with($lc_rightsValue,'access')">
```

Comment:
I'm a little worried about this as a solution.  First, does OpenAIREv4 require this field to be English language at all times?  If so, should we assume that any value ending in "access" is a valid OpenAIREv4 value, or should we specifically look for one of the four values listed at https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_accessrights.html

I'm not against the approach taken in this PR, but I just wonder if it solves the problem or not.  It looks, to me, like this value is *not* required to be English language, see: http://vocabularies.coar-repositories.org/documentation/access_rights/   It also looks like it's only allowed to be one of 4 different values.

Reply:
@tdonohue, I agree will what you are saying. Since [OpenAIRE 3 Access Level](https://guidelines.openaire.eu/en/latest/literature/field_accesslevel.html) is taken from dc:rights, that could be why [OpenAIRE 4 Access Rights](https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_accessrights.html) is taken from the same metadata field. As you mentioned, the better approach may be to use a different metadata field. @paulo-graca, should [openaire4-types.xml](https://github.com/DSpace/DSpace/blob/main/dspace/config/registries/openaire4-types.xml) be extended to include datacite:rights? If so, then that might be the preferred metadata field to use. The legacy use of dc:rights certainly does complicate matters given the large variety of content that can be in that metadata field.

In my initial [comment](https://github.com/DSpace/DSpace/issues/3097#issuecomment-747027882) I also suggested checking for all possible exact values since checking for ' access' at the end was an imperfect solution. It was a quick-win in our environment though (considering our existing metadata) and simplified the XSLT.

As for supporting multiple languages, what is the correct approach in DSpace in the context of OAI crosswalks?

Reply:
As is, like with [OpenAIRE 3 support](https://github.com/DSpace/DSpace/blob/main/dspace/config/crosswalks/oai/transformers/openaire.xsl#L31-L53), there is a pre-transformation process to normalize/translate existing `dc:rights` values to OpenAIRE 4 [supported values](https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_accessrights.html#definition-and-usage-instruction) that will only be used in this OAI context, doesn't affect whatever the repository already has stored.

I understand the need to detached it from English language and the easier solution could be to create a specific metadata field (we proposed that initially with OpenAIRE4 schema but agreed to use as much existing fields as possible), or even store the PURL value in `dc:rights`. But that would also have implications at the front-end and the end-user expectations, like:

- the value wouldn't be displayed out-of-the-box if we use a different field. Changes to DSpace+Angular would be required to enabled it to be "OpenAIRE 4" aware. And the requirement is kind of the opposite, DSpace 7 should be ready, but OpenAIRE4 should be an opt in feature. The admin must [install it](https://wiki.lyrasis.org/display/DSDOC7x/OpenAIRE4+Guidelines+Compliancy).
- using PURL/URI in `dc:rights`, without no changes to Angular, wouldn't give much to users. We could use [pipes](https://angular.io/guide/pipes) for translation, but with that, some business logic would be brought to the front-end and a full repository `dc:rights` translation process would be required to normalize all pre-existing metadata values. And considering the multiplicity of possibilities for `dc:rights` usage within DSpace community, I don't think that it's feasible. For instance in Portuguese the translation for "open access" is "acesso aberto" and we use and validate "openAccess" or "info:eu-repo/semantics/openAccess" for a `dc:rights` value for our repositories. We also have a translation mechanism at the presentation layer to display it differently to the user (with open access's iconography). But this is a specific implementation for RCAAP (that represents only 50% of all repositories in Portugal).

That said, I think the solution in place is the best possible for what it intends to achieve (despite some required adjustments). It doesn't mess with what the repository already has; It's an opt in feature and flexible enough to be extended/changed by anyone who looks to enhance it.

Reply:
@tdonohue please check this transformation: https://github.com/DSpace/DSpace/blob/main/dspace/config/crosswalks/oai/transformers/openaire4.xsl#L79-L114 it's a pre normalization for work. That's why you will have "access" English wording.

Reply:
@paulo-graca : Oh, I see now. I didn't realize that normalization is happening to use the English term "access"..  Could we reference that normalization from this section of the `oai_openaire.xsl` file then?  Since it's being pre-normalized in this way, I'm OK with this approach.  It just wasn't immediately evident to me...which is why a comment here could help!  The comment could just says something like:

`We are checking to ensure that only values ending in "access" can be used as datacite:rights.  This is a valid solution as we pre-normalize dc.rights values in openaire4.xsl to end in the term "access" according to COAR Controlled Vocabulary`